lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java

```diff
@@ -7449,11 +7441,7 @@ public String updateComments() {
 			// Create or update a task
 			String reference = "/lessonbuilder/item/" + comment.getId();
 			String title = getPage(comment.getPageId()).getTitle() + " - " + messageLocator.getMessage("simplepage.comments-task-title");
-			if (this.commentsCreateTask) {
-				createOrUpdateTask(reference, getCurrentSiteId(), title, null);
-			} else {
-				updateTask(reference, title, null);
-			}
+			createOrUpdateTask(reference, getCurrentSiteId(), title, null);
```

Comment:
Could we add a comment here?
// Dashboard task widget:  When a comment item is added create a task item for the site members.